devicetypes/smartthings/zwave-water-valve.src/zwave-water-valve.groovy

```diff
@@ -54,9 +56,18 @@ metadata {
 }
 
 def updated() {
+	// Device-Watch simply pings if no device events received for 32min(checkInterval)
+	unschedule()
+	runEvery15Minutes("healthPoll")
```

Comment:
WakeUp is only available on sleepy devices.

We should just get rid of this healthPoll thing. The whole point of ping() was for devices like this that don't check in regularly or need to be polled for status.